app/src/main/java/com/example/banking/domain/use_case/AccountsUseCases.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.banking.domain.use_case
+
+data class AccountsUseCases(
```

Comment:
why use case is data class? Data classes need to save data objects. Use case is a unit of logic and I can't see invoke method. If you have any complex logic you can combine use cases into one Interactor. But this Interactor must have business logic method which will call use cases methods and combine their result for example. Your solution doesn't make sense for me.